src/pages/index.js

```diff
@@ -1,118 +1,273 @@
+// API + userInfo
+// =============================================================================
+
+let userID;
+let cardList;
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "bcc6c959-bae4-4de1-885c-059746718b41",
+    "Content-Type": "application/json",
+  },
+});
+
+const userInfo = new UserInfo({
+  nameSelector: ".profile__title",
+  professionSelector: ".profile__description",
+  avatarSelector: ".profile__image",
+});
+
+Promise.all([api.getUserInfo(), api.getInitialCards()])
+  .then(([user, initialCards]) => {
+    userInfo.setUserInfo(user);
+    userID = user._id;
+    userInfo.setUserInfo(user);
+
+    cardList = new Section(
+      {
+        items: initialCards,
+        renderer: ({ name, link, isLiked, _id, userId, ownerId }) => {
+          const newCard = createCard({
+            name,
+            link,
+            isLiked,
+            _id,
+            userId,
+            ownerId,
+          });
+          cardList.addItem(newCard);
+        },
+      },
+      ".cards__list"
+    );
+    cardList.renderItems();
+  })
+  .catch(console.error);
+
+// =============================================================================
+// Handle Clicks
+// =============================================================================
+
+function handleLikeClick(cardInstance) {
+  if (cardInstance._likes) {
+    api
+      .unlikeCard(cardInstance._id)
+      .then((res) => {
+        cardInstance.setLikes(res.isLiked);
+      })
+      .catch(console.error);
+  } else {
+    api
+      .likeCard(cardInstance._id)
+      .then((res) => {
+        cardInstance.setLikes(res.isLiked);
+      })
+      .catch(console.error);
+  }
+}
+
+function handleCardClick(data) {
+  previewImagePopup.open(data);
+}
+
+const deleteCardPopup = new PopupWithConfirmation("#delete-card-modal");
+deleteCardPopup.setEventListeners();
+
+function handleDeleteClick(card) {
+  deleteCardPopup.open();
+  deleteCardPopup.setSubmitAction(() => {
+    deleteCardPopup.setLoading(true);
+    api
+      .deleteCard(card.getId())
+      .then(() => {
+        card.remove();
+        deleteCardPopup.close();
+      })
+      .catch(console.error)
+      .finally(() => {
+        deleteCardPopup.setLoading(false, "Yes");
+      });
+  });
+}
+
+function createCard({ name, link, isLiked, _id, userId, ownerId }) {
+  const cardElement = new Card(
+    { name, link, isLiked, _id, userId, ownerId },
+    userID,
+    "#card-template",
+    handleCardClick,
+    handleDeleteClick,
+    handleLikeClick
+  );
+  return cardElement.getView();
+}
+// =============================================================================
+// Preview Popup
+// =============================================================================
+
+const previewImagePopup = new PopupWithImage({
+  popupSelector: "#preview-modal",
+  handleFormSubmit: handleProfileEditSubmit,
```

Comment:
`handleFormSubmit: handleProfileEditSubmit,` is not needed for the **image modal**